notebooks/ImvizExample.ipynb

```diff
@@ -497,8 +499,8 @@
    "source": [
     "# Add 100 randomly generated X,Y (0-indexed) w.r.t. reference image\n",
     "# using default marker properties.\n",
-    "t_xy = Table({'x': np.random.randint(0, 4096, 100),\n",
-    "              'y': np.random.randint(0, 2048, 100)})\n",
+    "t_xy = Table({'x': np.random.randint(0, 14000, 50),\n",
```

Comment:
We might have to take the markers stuff out or delete the subsets first before doing this. It is taking too long on my machine with these new images.

Reply:
Reduced this to 10 markers but still too slow on my machine.